yolo-object-detection.html

```diff
@@ -3,7 +3,8 @@
         category: 'function',
         color: '#a6bbcf',
         defaults: {
-            detectionServerPort: {value:"8885"}
+            detectionServerPort: {value:"8885"},
+            name: {value:""}
```

Comment:
Oh dang this is why renaming didn't work, I noticed this and then forgot about it, thanks for fixing.